src/fluree/server/config.clj

```diff
@@ -193,4 +204,6 @@
   ([path profile]
    (-> path
        read-file
-       (finalize profile))))
+       (cond->
+        (= profile :task) (task-config profile)
```

Comment:
This overloads the concept of a profile. A profile is meant to override values in a pre-defined configuration, and a task is a wholly new configuration. Is there an issue with a user supplying a separate configuration file when they want to run a migration?